crates/fuel-gas-price-algorithm/gas-price-analysis/src/charts.rs

```diff
@@ -76,19 +76,29 @@ pub fn draw_gas_prices(
     da_gas_prices: &[u64],
     title: &str,
 ) -> anyhow::Result<()> {
+    let gas_prices_gwei: Vec<_> =
+        gas_prices.into_iter().map(|x| x / 1_000_000_000).collect();
+    let _exec_gas_prices_gwei: Vec<_> = _exec_gas_prices
+        .into_iter()
+        .map(|x| x / 1_000_000_000)
```

Comment:
nit: suggestion to not use magic numbers, but this works fine as well :)
```suggestion
        .map(|x| x / ONE_GWEI)
```